packages/varlet-ui/src/styles/common.less

```diff
@@ -97,6 +97,8 @@
   --color-surface-container-highest: hsla(var(--hsl-surface-container-highest), 1);
   --hsl-inverse-surface: 0, 0%, 20%;
   --color-inverse-surface: hsla(var(--hsl-inverse-surface), 1);
+  --hsl-filled: 0, 0%, 93%;
+  --color-filled: hsla(var(--hsl-filled), 1);
```

Comment:
We don't need filled variable in common. I suggest that it only be in the card component.